.github/workflows/release-suite-desktop-web-production.yml

```
@@ -65,12 +65,26 @@ jobs:
         with:
           role-to-assume: arn:aws:iam::538326561891:role/gh_actions_suite_production
           aws-region: eu-central-1
+      - name: Extract suite desktop version
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2034:warning:1:1: VERSION appears unused. Verify use (or export if used externally)

(shellcheck)